### PR TITLE
toxcore-git: update to 20151211.

### PR DESCRIPTION
**Pull request by Spencer-H**

### CODE REVIEW

File: srcpkgs/toxcore-git/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Template file for 'toxcore-git'
 pkgname="toxcore-git"
-version="20150604"
-revision=1
-_commithash="c259250abb86a01ae7e3fc906380417427f38ca1"
+version="20151211"
+revision=2
```
Comment (xtraeme) [inline srcpkgs/toxcore-git/template]: `revision` must be set to 1 for new packages/updates.

**Comment by Spencer-H** [comment]
Yeah. Sorry I had that changed locally to force an update over the version I was using. I forgot to change it back before pushing.

**Comment by Duncaen** [comment]
You can use the `-f` flag to force build/install with `xbps-src` and `xbps-install`.

**Comment by Spencer-H** [comment]
Ah. I didn't know you could use that with xbps-install, that was the problem I was having. It was seeing it was the same version and using the cached version instead.

**Pull request merged**